dnf-behave-tests/common/file.py

```diff
@@ -47,9 +47,8 @@ def step_impl(context, filepath):
 @behave.given('I create symlink "{dst}" to file "{src}"')
 def step_impl(context, dst, src):
     dst = prepend_installroot(context, dst)
-    src = prepend_installroot(context, src)
     ensure_directory_exists(os.path.dirname(dst))
-    os.symlink(src, dst)
+    os.symlink(src.format(context=context), dst)
```

Comment:
I think we should be consistent in how we handle the paths in the file manipulation steps. I'm already pretty confused about what steps do substitution and where and what steps prepend the installroot. Not necessarily in this PR, but for this one function... is there a problem with both prepending the installroot and doing the substitution? I'd just do both on all the paths in all the steps :slightly_smiling_face: (so, e.g. here I'd do it on destination too - perhaps we'd just have to check if it's a relative path and don't prepend installroot for that case)

Reply:
I didn't realize that the prepend_installroot function already does this, it can prepend installroot and do the substitution as well. So I can undo my change and just prepend all the paths with `/`. 

Since https://github.com/rpm-software-management/ci-dnf-stack/commit/0bfed1f6e418800b8e19e7c9d73fc350b1e584db I can just do `/{context.scenario.repos_location/path/to/createrepo_c/repo` and the prepend installroot will do the substitution and since it will have two `//` it will not prepend the installroot.

Is this what you mean by doing both?

Reply:
Yeah, that's what I meant. And yeah, it does indeed do that already, so you can use that :slightly_smiling_face:

Reply:
To answer your question I don't think it makes much sense to try to do both at the same time. Prepending installroot while doing substitution on `{context.scenario.repos_location}` does not seem helpful because repos_location is not inside installroot.